source/lib/position/getOffsets.js

```diff
@@ -0,0 +1,32 @@
+import { POSITION } from './constants/position';
+import { reposition } from './modifiers/reposition';
+import { getOffsetsObject, getOffScreenValidationObject } from './utils';
+
+/**
+ * @function
+ * @argument {Element} offsetElement - the element to get offsets for
+ * @argument {Element} targetElement - the element that is being offset from
+ * @argument {POSITION} position - (Optional) the position of the offset element
+ * @returns {Object} the offsets object, containing position
+ */
+function getOffsets (offsetElement, targetElement, position = POSITION.BOTTOM_RIGHT) {
+    let offsets = getOffsetsObject(position, offsetElement, targetElement);
+    let offscreen = getOffScreenValidationObject(offsetElement, offsets);
+
+    //If the element is offscreen, reposition the element
+    if (offscreen.anywhere) {
+        let newPosition = reposition(position, offscreen);
```

Comment:
Move all of this logic (as-is) to `source/lib/position/index.js`.

Current logic in `source/lib/position/index.js` is redundant, because it's literally exporting the same object this file is exporting.